src/shogun/converter/StochasticProximityEmbedding.h

```diff
@@ -102,31 +104,31 @@ class StochasticProximityEmbedding : public EmbeddingConverter
 		 *
 		 * @param tolerance regularization value
 		 */
-		void set_tolerance(float32_t tolerance);
+		void set_tolerance(float64_t tolerance);
 
 		/** getter for regularization parameter
 		 *
 		 * @return regularization value
 		 */
-		int32_t get_tolerance() const;
+		float64_t get_tolerance() const;
 
 		/** setter for number of updates per iteration
 		 *
-		 * @param nupdates number of updates per SPE iteration
+		 * @param num_updates number of updates per SPE iteration
 		 */
-		void set_nupdates(int32_t nupdates);
+		void set_num_updates(int32_t num_updates);
 
 		/** getter for number of updates per iteration
 		 *
-		 * @return nupdates value
+		 * @return num_updates value
 		 */
-		int32_t get_nupdates() const;
+		int32_t get_num_updates() const;
 
 		/** setter for the maximum number of iterations
 		 *
 		 * @param max_iteration the maximum number of iterations
 		 */
-		void set_max_iteration(const int32_t max_iteration);
+		void set_max_iteration(int32_t max_iteration);
```

Comment:
++

Reply:
Was the const it causing issues?

Reply:
Clang-tidy says: https://clang.llvm.org/extra/clang-tidy/checks/readability-avoid-const-params-in-decls.html

Reply:
I see. OK!